bundles/org.codechecker.eclipse.plugin/src/org/codechecker/eclipse/plugin/codechecker/ICodeChecker.java

```diff
@@ -49,7 +49,7 @@
      *            The configuration being used.
      * @return The analyze command as String.
      */
-    public String getAnalyzeString(CcConfigurationBase config, @Nullable Path logFile);
+    public String getAnalyzeString(CcConfigurationBase config, @Nullable Path logFile, @Nullable Path skipFile);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected @param tag for 'skipFile'.](https://app.codacy.com/app/CodeChecker/CodeCheckerEclipsePlugin/pullRequest?prid=4641620)